src/components/OrderHistory/OrderListItem.tsx

```diff
@@ -29,7 +29,7 @@ const OrderListItem = ({
           주문 날짜: 2024-08-13 <br />
           픽업장소: {pickUpLocation}
           <br />
-          결제금액: {price}P
+          {price != null ? `결제금액: ${price}P` : ``}
```

Comment:
`!=` 연산자로 비교를 하신 이유가 있을까요? `undefined`와 `null`을 모두 검증하려는 목적이었다면 아래처럼 간단하게 코드를 작성해도 괜찮을거 같네용

```suggestion
          {price ? `결제금액: ${price}P` : ``}
```

Reply:
아 혹시 `0`이 들어왔을 때는 결제금액을 표시해주기 위함이었나요?

Reply:
아 저런 방법이 있다는걸 생각 못했습니다..!